rust/libnewsboat/src/utils.rs

```diff
@@ -87,6 +88,32 @@ pub fn absolute_url(base_url: &str, link: &str) -> String {
         .to_owned()
 }
 
+pub fn resolve_tilde(path: String) -> String {
+    let mut file_path:String = path;
+    let home_path = dirs::home_dir();
+
+    if home_path.is_some() {
+        let home_path_string = home_path.unwrap().display().to_string();
```

Comment:
1. Please avoid `unwrap()`, because it will crash the whole program. In this case, you can replace `if home_path.is_some()` with `if let Some(home_path) = home_path`, and remove the `unwrap()`.
2. `display().to_string()` works, but its semantics is more like "give me a string to display to the user". I think `to_string_lossy().into_owned()` would be better here. The only *real* argument I have is that `to_string_lossy()` clearly documents how it modifies the string, while `display()` doesn't.